wordle-web/pages/wordleOfTheDay.vue

```diff
@@ -0,0 +1,167 @@
+<template>
+  <UserNameDialog v-model="showUserNameDialog" />
+
+  <HelpDialog v-model="showHelpDialog" />
+  <v-container>
+    <v-btn icon="mdi-help-box" @click="showHelpDialog = true" />
+    <v-progress-linear v-if="game.isBusy" color="primary" indeterminate />
+    <v-card v-else class="text-center">
+      <v-alert
+        v-if="game.gameState != GameState.Playing"
+        :color="game.gameState == GameState.Won ? 'success' : 'error'"
+        class="mb-5"
+        tile
+      >
+        <h3>
+          You've
+          {{ game.gameState == GameState.Won ? "Won!" : "Lost :()" }}
+        </h3>
+        <v-card-text>
+          The word was: <strong>{{ game.secretWord }}</strong>
+        </v-card-text>
+        <v-row v-if="game.stats" class="mb-1" justify="center">
```

Comment:
I like the use of the progress circular!!

Reply:
Thank you!